css/src/tokens/typography.scss

```diff
@@ -0,0 +1,21 @@
+// Typography
+$font-size-9: 0.75rem !default; // 12px
+$font-size-8: 0.875rem !default; // 14px
+$font-size-7: 1rem !default; // 16px
+$font-size-6: 1.125rem !default; // 18px
+$font-size-5: 1.25rem !default; // 20px
+$font-size-4: 1.5rem !default; // 24px
+$font-size-3: 1.75rem !default; // 28px
+$font-size-2: 2.125rem !default; // 34px
+$font-size-1: 2.5rem !default; // 40px
+$font-size-0: 3.375rem !default; // 54px
+
+// Base
```

Comment:
Let's use the following variable names:

```scss
$font-size-jumbo
$font-size-large
$font-size-body
$font-size-small
$font-size-extra-small
$font-size-tiny
```

Reply:
lets add them , once we have definitive sizes for them .

Reply:
On ACOM we used to use words like "small", "extra-small", and "tiny" but it became difficult for people to understand if "extra-small" was smaller than "tiny", or vice-versa, on the name alone.

We did switch to numeric sizing "size-1", "size-2", "size-3", etc. which removed the ambiguity and has worked for us.

Alternatively, if a non-numeric value is desired, following the t-shirt sizing model could help: "small", "xsmall", and "xxsmall".

Reply:
agreed- numeric scale is as meaningful as t-shirt sizes and familiar (heading elements use numeric scale).

Reply:
ah - @wibjorn reminded me this is the existing scale/names. So we'll need to track some work to make this consistent with whatever the new pattern is coming out of the spacing work. We definitely want to account for @squalrus's comment about ambiguity.

Reply:
It's worth noting there is actual a fair bit in the air regarding how we'll structure our atomics. Next week we should be able to solidify the team's opinions regarding that. The semantic sizes I've put above (while needed to make docs-ui truly inherit from Atlas) don't need to be included until we've figured that structure.

We did just make in this pattern Docs for font scale. (Aside: I think we could remove `tiny`, so it's not competing with extra-small), and it includes a mixture of two categories of atomic class for font-size: 1) those pointed directly headings and 2) those for more general non-heading use cases (large, body, small, etc). We can revisit this next week though.